### PR TITLE
Fixes for ShellCheck errors

### PR DESCRIPTION
**Pull request by atrepca**
Running the script through the [shellcheck](https://github.com/koalaman/shellcheck) static analysis tool returns the errors below:
```sh
$ shellcheck run.sh 

In run.sh line 1:
#!/usr/bin/with-contenv bashio
^-- SC1008 (error): This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh. Add a 'shell' directive to specify.


In run.sh line 17:
export AWS_ACCESS_KEY_ID="$(bashio::config 'aws_access_key')"
       ^---------------^ SC2155 (warning): Declare and assign separately to avoid masking return values.


In run.sh line 18:
export AWS_SECRET_ACCESS_KEY="$(bashio::config 'aws_secret_access_key')"
       ^-------------------^ SC2155 (warning): Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC1008 -- This shebang was unrecognized. Sh...
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

SC2155 is detailed [here](https://www.shellcheck.net/wiki/SC2155), and SC1008 [here](https://www.shellcheck.net/wiki/SC1008).

ShellCheck can also run as a [GitHub action](https://github.com/marketplace/actions/shellcheck).

Also removed the executable bit from other non-executables that had it.

### CODE REVIEW

File: amazon-s3-backup/run.sh [hunk 1]
```diff
@@ -14,8 +15,8 @@ local_backups_to_keep="$(bashio::config 'local_backups_to_keep' '3')"
 monitor_path="/backup"
 jq_filter=".backups|=sort_by(.date)|.backups|reverse|.[$local_backups_to_keep:]|.[].slug"
 
-export AWS_ACCESS_KEY_ID="$(bashio::config 'aws_access_key')"
-export AWS_SECRET_ACCESS_KEY="$(bashio::config 'aws_secret_access_key')"
+AWS_ACCESS_KEY_ID="$(bashio::config 'aws_access_key')"; export AWS_ACCESS_KEY_ID
```
Comment (thomasfr) [inline amazon-s3-backup/run.sh]:
```suggestion
AWS_ACCESS_KEY_ID="$(bashio::config 'aws_access_key')"
export AWS_ACCESS_KEY_ID
```

File: amazon-s3-backup/run.sh [hunk 2]
```diff
@@ -14,8 +15,8 @@ local_backups_to_keep="$(bashio::config 'local_backups_to_keep' '3')"
...
+AWS_SECRET_ACCESS_KEY="$(bashio::config 'aws_secret_access_key')"; export AWS_SECRET_ACCESS_KEY
```
Comment (thomasfr) [inline amazon-s3-backup/run.sh]:
```suggestion
AWS_SECRET_ACCESS_KEY="$(bashio::config 'aws_secret_access_key')"
export AWS_SECRET_ACCESS_KEY
```

**Review by thomasfr — CHANGES_REQUESTED** [review]
Thank you very much. Did not know shellcheck tbh and was not aware of SC2155. So thanks for that. 
If you or someone else interested in contributing, i would also accept a PR for enabling ShellCheck within Github Actions.

**Comment by atrepca** [comment]
Hi @thomasfr, I updated this to add new lines as requested, thanks.

**Comment by atrepca** [comment]
Going to close this and fix the ShellCheck errors in #20 so the action starts with a clean state.

**Pull request closed without merging**